.github/workflows/build_docs.yaml

```diff
@@ -120,7 +120,7 @@ jobs:
                       docs/quickstart/cpp/book/html
                       docs/quickstart/node/book/html
             - name: "Check for docs warnings in internal crates"
```

Comment:
There is no internal docs for the slint-python crate, and that's why there is no point of checking for warnings?

Reply:
I'm fine with it. No need to revert.
I usually like to have internal docs for anything. But I wouldn't generate it anyway.
And these internal docs might contains warnings that no one would anyway see since it's unlikely that they generate the docs for it.